cmd/fleetagent/main.go

```diff
@@ -21,6 +21,7 @@ var (
 type FleetAgent struct {
 	Kubeconfig      string `usage:"kubeconfig file"`
 	Namespace       string `usage:"namespace to watch" env:"NAMESPACE"`
+	AgentScope      string `usage:"An identifier used to scope the agent bundleID names, typically the same as namespace" env:"AGENT_SCOPE"`
```

Comment:
Why "scope" over "ID"? I'm confused about what "scope" means in this context.

Reply:
AgentScope == labelSuffix, so read the comment above about what labelSuffix does.